### DISCUSSION TITLE
[Announcement] Homework 3 is out

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs524-program-analysis#321] [Announcements]
Hi all,

This is an announcement about Homework 3.
Here is the GitHub classroom link for this homework:
https://classroom.github.com/a/pvmFyvgP

## Submission
You should submit your answers in both GitHub and Gradescope.
Write your answer in the latex template file, and push your commit to GitHub.
Submit the compiled pdf file to Gradescope.
**DO NOT** change the problem region defined in Gradescope system.
Please refer to the README.md of the homework repository.

## Due date
Due: **10. 07 (Sat.) 23:59:59**
Late Due: **10. 09 (Mon.) 23:59:59** (late submission penalty).

**Comment by Re-st** [comment]
Can we write in Korean if preferred, like in HW1?

**Comment by KihongHeo** [comment]
Please write in Korean if you are a native Korean.
